common/frame_manager.go

```diff
@@ -390,10 +390,16 @@ func (m *FrameManager) frameRequestedNavigation(frameID cdp.FrameID, url string,
 	return nil
 }
 
-func (m *FrameManager) getFrameByID(id cdp.FrameID) *Frame {
+// getFrameByID will attempt to find a frame with the given id. If one is found
+// the frame and true will be returned. If one isn't found nil and false will
+// be returned.
+func (m *FrameManager) getFrameByID(id cdp.FrameID) (*Frame, bool) {
 	m.framesMu.RLock()
 	defer m.framesMu.RUnlock()
-	return m.frames[id]
+
+	frame, ok := m.frames[id]
+
+	return frame, ok
```

Comment:
We use `ok` here, and `found` in other parts of the code. It might be better to use `ok` in the rest, too, as it's also idiomatic to use `ok`.

Reply:
Yeah, i was wondering whether `found` read a bit better. Happy to change it ok.

Reply:
Refactored in 5bb8c607033917d1ce243a921c8ca48451eedb6b